index.d.ts

```diff
@@ -404,3 +402,5 @@ declare class Long {
      */
     xor(other: Long | number | string): Long;
 }
+
+export { Long };
```

Comment:
Does the type definition still support `import * as Long from "long"` for backwards compatibility?

Reply:
Nope it does not. Just saw the type definition in `@types/long` package. If people are already using that then this will be a breaking change. Will keep the same design for backwards compatibility.